### PR TITLE
SOT-26: Write chapter 3 of Proofs, Arguments, and Zero-Knowledge

### PR DESCRIPTION
**Pull request by VanhGer**
- [ ] Did you check your grammar / reader proof your document using ChatGPT?
- [ ] Don't forget to squash commits into meaningful chunks before merging
- [ ] After merging this, please update your task status!

### CODE REVIEW

**Review by ktokto313 — APPROVED** [review]
I think the obsidian part should be in git-ignore section, it's doesn't provide any additional value to the commit.

**Comment by VanhGer** [comment]
> I think the obsidian part should be in git-ignore section, it's doesn't provide any additional value to the commit.

Mr Son said that he wanted us to have the same plugins of this app. So he decided not to remove .obsidian folder.

**Review by Draply — APPROVED** [review]

File: articles/proofs-arguments-and-zero-knowledge/chapter_3.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+- Completeness: For any $x \in L$, there are some prover strategies that will cause the verifier to accept with high probability. 
+- Soundness: For any $x \notin L$, then for every prover strategy, the verifier rejects with high probability.
+The above properties do not require that *false statement*s (which inputs not in the language) have convincing refutations (i.e., convincing proofs of their falsity)
+
+**NP and IP**
+
+**IP**: the class of all languages solvable by an interactive proof system with a polynomial time verifier.
+**NP**: IP, but it is non-interactive and deterministic, and completeness and soundness errors are 0.
+
+## 3.4 Schwartz-Zippel Lemma
```
Comment (sonntuet1997) [inline articles/proofs-arguments-and-zero-knowledge/chapter_3.md:36]: Should write this in a separate file

File: articles/proofs-arguments-and-zero-knowledge/chapter_3.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+**Lemma 3.3 (Schwartz-Zippel Lemma)**
+Let $F$ be any field, and let g: $F^m → F$ be a nonzero $m$-variate polynomial of total degree at most $d$. Then on any finite set $S \subseteq F$ : $$Pr_{x←S^m} [g(x) = 0] ≤ d/|S|.$$
+## 3.5 Low Degree and Multilinear Extensions
```
Comment (sonntuet1997) [inline articles/proofs-arguments-and-zero-knowledge/chapter_3.md:40]: This also

Comment (VanhGer) [reply articles/proofs-arguments-and-zero-knowledge/chapter_3.md:40]: Sure. I will update soon

**Pull request closed without merging**